lib/reporting/reports/orders_and_fulfillment/base.rb

```diff
@@ -28,6 +28,10 @@ def query_result
           }.values
         end
 
+        def metadata_option_opt_in
+          true
+        end
```

Comment:
The metadata builder is already finding the metadata without much knowledge about the report. Could it also decide which report has metadata to display? Then you don't need these methods.